examples/python/ui/main.js

```diff
@@ -0,0 +1,122 @@
+const SERVICE_URL = 'http://iam.local:8000'
+
+const AUTH0_CLIENT_ID = 'SLocf7Sa1ibd5GNJMMqO539g7cKvWBOI';
+const AUTH0_DOMAIN = 'auth.mozilla.auth0.com';
+const AUTH0_CALLBACK_URL = window.location.href;
+const API_AUDIENCE = AUTH0_CLIENT_ID;
+const SCOPES = 'openid groups email';
+
+
+document.addEventListener('DOMContentLoaded', main);
+
+function main() {
+  const logoutBtn = document.getElementById('logout');
+  logoutBtn.addEventListener('click', logout);
+
+  const webAuth = new auth0.WebAuth({
+    domain: AUTH0_DOMAIN,
+    clientID: AUTH0_CLIENT_ID,
+    redirectUri: AUTH0_CALLBACK_URL,
+    audience: API_AUDIENCE,
```

Comment:
Does this work in the end?

Reply:
I don't think so but it should be a bug to file in the auth0 lib